CONTRIBUTING.md

```diff
@@ -113,10 +132,10 @@ If you are interested in creating a translation, here are some pointers for how
 * Often there is work to do in creating markdown files or updating the translation to keep it up to date with latest changes.
 * We would request that you base your translation on the 4.0/en folder in the master branch as this is now static at the 4.0.3 version.
 * In order to start a translation, please start by forking the ASVS repository.
-    * If you are updating an existing translation which has markdown (just French as at December 2021), you can make modifications to the files in the existing folder based on language code (just /fr as at December 2021).
-    * if you are starting a new markdown translation, take a copy of the /en folder and rename it to the 2 character language code which will be used for the translation. 
+    * If you are updating an existing translation which has markdown (just French as of December 2021), you can make modifications to the files in the existing folder based on language code (just /fr as at December 2021).
+    * If you are starting a new markdown translation, take a copy of the /en folder and rename it to the 2 character language code which will be used for the translation.
 * When you have completed the translation, please open a Pull Request against the master ASVS branch and one of the leaders will look at integrating it.
-* The leader will also use the relevant scripts to create the documents from the raw markdown (or you can if you want to save us some trouble  )
-* Finally, the leader will back port the translation into the branch containing ASVS version which was targeted (at this point, presumably v4.0.3)
+* The leader will also use the relevant scripts to create the documents from the raw markdown (or you can if you want to save us some trouble).
+* Finally, the leader will back port the translation into the branch containing the ASVS version which was targeted (at this point, presumably v4.0.3).
```

Comment:
Need to change the version

Reply:
Well for now it will be 4.0.3 until we have a solid 5.0